Problem3.asm

```diff
@@ -0,0 +1,31 @@
+#Justin Hiller
+#Comp 3410
+#PA[1] problem 3
+#February 11, 2015
+#This program calculates the sum of the odd integers(inclusive) between 1-15
+
+	.data
+intro:		.asciiz		"The sum of the odd integers between 1-15 inclusive is: "
+
+	.text
+	
+	la $a0, intro		#Prints the introduction
+	li $v0, 4
+	syscall
+	
+	la $s0, 1		#loads the number 1 into s0
```

Comment:
-2 Line 16:
	Should be using 'li' if you want to load a literal number (as you did on line 13, 26, and 29).  It works but its bad style.